File: js_ts/variaveis/ativi1/saldo.ts
const saldo_inicial: number = 2000.00

const nome = String(prompt("seu nome:"))
const valor_depositado = Number(prompt("insira o valor depositado:"))
const valor_sacado = Number(prompt("insira o valor sacado:"))

const saldo_final: number = saldo_inicial + valor_depositado - valor_sacado

console.log("cliente de nome:" , nome)
console.log("saldo atual:" , saldo_final)

File: python/variaveis/atividades1/saldo.py
nome = "wesllen"
saldo = 2300.50

valorDepositado = int(input("quanto vc quer depositar? "))
valorSacado = int(input("quanto vc quer retirar? "))

saldo = saldo + valorDepositado - valorSacado

print("cliente de nome:", nome)
print("saldo atual:", saldo)
